lib/wx/wxwin-nothrow.m4

```diff
@@ -321,32 +321,7 @@ AC_DEFUN([WX_CONFIG_CHECK],
        WX_LIBS=""
        WX_LIBS_STATIC=""
        WX_RESCOMP=""
-
```

Comment:
I don't understand why this is needed it worked before?

Reply:
Yes I agree this is a different effort. I'll remove from this from this PR and put into a different PR. Honestly I really don't have any opinion here. Just tried to match @garazdawi explanation and compared a bit with the lib/odbc/configure.in but not sure whether that's a prime example either. 

What stopped the builds was the error here: `AC_MSG_ERROR([$wx_error_message])`
Even if `--with-wx` was not provided, so that's the reason I removed it.

Reply:
Moved https://github.com/erlang/otp/pull/4518/